packages/react-sdk/src/components/html.tsx

```diff
@@ -0,0 +1,19 @@
+import { forwardRef, type ElementRef } from "react";
+
+type Props = {
+  code?: string;
+};
+
+export const Html = forwardRef<ElementRef<"div">, Props>((props, ref) => {
+  const { code, ...rest } = props;
+  return (
+    <div
+      {...rest}
+      ref={ref}
+      style={{ display: code?.trim().length !== 0 ? "contents" : "block" }}
```

Comment:
why not using data-state attribute and based on it style over preset?

Reply:
actually you don't even need to ever use block, because when there is no user code pasted we can render fallback that will hold the spacing